sources/packages/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -297,6 +298,7 @@ export class OfferingValidationModel {
   @IsDateString(undefined, {
     message: getDateFormatMessage(userFriendlyNames.studyEndDate),
   })
+  @IsDateAfter(studyStartDateProperty, userFriendlyNames.studyEndDate)
```

Comment:
we don't care if both dates are the same right? the end should always be after the start right?

Reply:
That was the intention 😉